src/main/java/com/internet/shop/controllers/RegistrationController.java

```diff
@@ -0,0 +1,40 @@
+package com.internet.shop.controllers;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.models.User;
+import com.internet.shop.services.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class RegistrationController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private UserService userService = (UserService) injector.getInstance(UserService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/user/register.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+
+        String name = req.getParameter("name");
+        String login = req.getParameter("login");
+        String psw = req.getParameter("psw");
+        String pswRepeat = req.getParameter("psw-repeat");
+
+        if (psw.equals(pswRepeat)) {
+            User user = new User(name, login, psw);
+            userService.create(user);
```

Comment:
let's create a shopping cart for the user during registration

Reply:
added